modules/mod_login/Helper/LoginHelper.php

```diff
@@ -63,4 +63,33 @@ public static function getType()
 
 		return (!$user->get('guest')) ? 'logout' : 'login';
 	}
+
+	/**
+	 * Retrieve the URL for the registration page
+	 *
+	 * @return  string
+	 */
+	public static function getRegistrationUrl($params)
+	{
+		$regLink = 'index.php?option=com_users&view=registration';
+		$regLinkMenuId = $params->get('customRegLinkMenu', 0);
```

Comment:
Default value in XML is empty string. You can use it as fallback or use `null` or just omit the value, e.g.
```suggestion
		$regLinkMenuId = $params->get('customRegLinkMenu');
```